docs/datasets.md

```diff
@@ -234,3 +234,22 @@ The structure of the key information extraction dataset directory is organized a
 ```
 
 - Download [wildreceipt.tar](https://download.openmmlab.com/mmocr/data/wildreceipt.tar)
+
+
+## Named Entity Recognition
+### CLUENER2020
+The structure of the named entity recognition dataset directory is organized as follows.
+
+```text
+cluener_public
```

Comment:
Change the style, similar to others

![image](https://user-images.githubusercontent.com/9464825/117914173-5bc11e00-b315-11eb-83fb-55c6f70a60e9.png)